easybuild/framework/easyblock.py

```diff
@@ -1068,8 +1067,9 @@ def make_module_footer(self):
         footer = [self.module_generator.comment("Built with EasyBuild version %s" % VERBOSE_VERSION)]
 
         # add extra stuff for extensions (if any)
-        if self.cfg['exts_list']:
-            footer.append(self.make_module_extra_extensions())
+        extensions = self.cfg['exts_list']
+        if extensions:
+            footer.append(self.make_module_extra_extensions(extensions))
```

Comment:
hmm, there's little point in passing down `self.cfg['exts_list']` here, since self.make_module_extra_extensions also has access to it?